__tests__/SetsSelectors.test.js

```diff
@@ -820,6 +820,286 @@ describe('SetsSelectors', () => {
         });
     });
 
+    describe('getBestAvgVelocityEver', () => {
+
+        test('get best avg velocity ever', () => {
```

Comment:
- [ ] modify test to have a larger value as invalid and another larger value as removed, should still work